### PR TITLE
Make some edits to README.md

### PR DESCRIPTION
**Pull request by pmorie**
Make a couple cleanups to README.md to:

1.  Communicate the mission without getting bogged down in terminology
2.  Add code of conduct (important!)

### CODE REVIEW

**Comment by arschles** [comment]
Giving my LGTM1 assuming @duglin's [comment](https://github.com/kubernetes-incubator/service-catalog/pull/152#discussion_r92001111) is somehow addressed

File: README.md [hunk 1]
```diff
@@ -2,83 +2,40 @@
...
 
-We are currently scoping our **v1** milestone [here](./docs/v1). Interested
-in contributing?  Check out the [documentation](./CONTRIBUTING.md)
+This is a [Kubernetes Incubator project](https://github.com/kubernetes/community/blob/master/incubator.md).
+The project was established 2016-Sept-12.  The incubator team for the project is:
 
-## Contributing
+- Sponsor: Brian Grant (@bgrant0607)
+- Champion: Paul Morie (@pmorie)
+- SIG: [sig-service-catalog](https://github.com/kubernetes/community/tree/master/sig-service-catalog)
```
Comment (duglin) [inline README.md]: ok, not sure how i feel about stuff that we control (slack channel info, weekly call times, ...) being in a repo that we don't control, but... if we do keep it there and not here, let's make it clear that people should follow the link. Saying just "SIG" wasn't obvious to me.
How about:
`More information on contributing and participating in the SIG:`

File: README.md [hunk 2]
```diff
@@ -2,83 +2,40 @@
...
+The service-catalog project is in incubation to bring integration with service
+brokers to the Kubernetes ecosystem.  A service broker is an endpoint that
+manages a set of services.  The end-goal of the service-catalog project is to
+provide a way for Kubernetes users to consume services from brokers and easily
+configure their applications to use those services.
 
-**Service Broker**: An endpoint that manages a set of services. Responsible for
-translating Service Catalog activities (like provision, bind, unbind, deprovision)
-into appropriate actions for the service.
+<hr>
```
Comment (MHBauer) [inline README.md]: raw html?

File: terminology.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+**Application**: Code that will access or consume a Service. While in Kubernetes
+the code that is deployed is often called a "Service", to avoid confusion, this
+document will refer to the code that accesses a service as an "application".
+
+**Resource type**: A logical Kubernetes concept. Examples include:
+
+  - [Pod](http://kubernetes.io/docs/user-guide/pods/)s
+  - [Service](http://kubernetes.io/docs/user-guide/services/)s
+  - [Secret](http://kubernetes.io/docs/user-guide/secrets/)s
```
Comment (MHBauer) [inline terminology.md]: is there any particular reason to not have the s?

it's weird looking.

File: README.md (hunk 2 above)
Comment (pmorie) [inline README.md]: yeah, I don't know how to do an hr in markdown, do you?  I was following the format of the Kubernetes readme.

File: README.md (hunk 1 above)
Comment (pmorie) [inline README.md]: @duglin The information belongs in kubernetes/community -- we can put any info we want there.

**Comment by pmorie** [comment]
@duglin @MHBauer nits addressed, PTAL

**Comment by duglin** [comment]
LGTM

File: README.md [hunk 4]
```diff
@@ -2,83 +2,40 @@
...
 
-We have a google mailing list: https://groups.google.com/forum/#!forum/kubernetes-sig-service-catalog
+## Code of Conduct
 
-We have a SIG Slack Channel: https://kubernetes.slack.com/archives/sig-service-catalog
+Participation in the Kubernetes community is governed by the
+[Kubernetes Code of Conduct](./code-of-conduct.md).
 
-We have a weekly call: https://plus.google.com/hangouts/_/google.com/k8s-sig-service-catalog   at 1pm PT on Mondays
-- And an agenda doc: https://docs.google.com/document/d/10VsJjstYfnqeQKCgXGgI43kQWnWFSx8JTH7wFh8CmPA/edit
```
Comment (MHBauer) [inline README.md]: Agree that the details should be in k8s/community. That repo has been useful to me to find out when and where a sig is, more than once.

**Comment by pmorie** [comment]
Nits addressed @MHBauer, thanks for the lgtmish anyway :)

**Comment by pmorie** [comment]
@bmelville @martinmaly @vaikas-google can I get a review / LGTM on this?

File: README.md [hunk 5]
```diff
@@ -2,83 +2,43 @@
 
 [![Build Status](https://travis-ci.org/kubernetes-incubator/service-catalog.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/service-catalog)
 
-### Terminology
+### Introduction
```
Comment (martinmaly) [inline README.md]: should this be ## ?

File: README.md [hunk 6]
```diff
@@ -2,83 +2,43 @@
...
-However, this SIG is (initially) focused on implementing integration with the
-[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
-Additionally, we will be exploring what can be done to help Service developers
-when they want to expose their code via a Service Broker.
+This is a [Kubernetes Incubator project](https://github.com/kubernetes/community/blob/master/incubator.md).
+The project was established 2016-Sept-12.  The incubator team for the project is:
 
-We are currently scoping our **v1** milestone [here](./docs/v1). Interested
-in contributing?  Check out the [documentation](./CONTRIBUTING.md)
+- Sponsor: Brian Grant (@bgrant0607)
```
Comment (martinmaly) [inline README.md]: make Brian link too?

**Comment by arschles** [comment]
I'm still LGTM as-is. Happy with fixing up any imperfections in a future PR

File: README.md (hunk 5 above)
Comment (pmorie) [inline README.md]: Yes, nice catch!

**Comment by pmorie** [comment]
@martinmaly nits addressed, thanks for the review.

**Comment by pmorie** [comment]
Travis build doesn't seem to have started yet.

**Comment by pmorie** [comment]
Since this is out of band relative to the build, do people object to a merge?  I'd really like to get this in since we are going to have a lot of traffic to the repo today.

**Pull request merged**